python/cudf/cudf/_lib/orc.pyx

```diff
@@ -22,6 +25,19 @@ import errno
 import os
 
 
+cdef unique_ptr[cudf_table] make_table_from_columns(columns):
```

Comment:
I did try that initially, but I was getting failures in other cudf pytests where the incoming columns were Series rather than a list of columns (https://github.com/rapidsai/cudf/pull/2507/commits/26d658412d18f07915effd5436cf32f62f85f989)

My Python wasn't good enough to resolve it without almost making two separate paths in the same functions. Any solutions?

Reply:
We can revisit this in a followup PR, one thing I notice is we're not setting the name of the gdf_column being created. Are you handling names elsewhere or should we be doing that?

Reply:
I think `column_view_from_column` transfers/sets the incoming column name?

Reply:
Yes it does, total brainfart, misread the function completely somehow!